docs/blog/20240612/20240612.md

```diff
@@ -0,0 +1,201 @@
+# Introduction to 5G Quality of Service (QoS)
+
+>[!NOTE]
+> Author: TingYuan Chou
+> Date: 2024/06/12
+---
+
+## Introduction
+
+Quality of Service refers to the performance characteristics of a network orservice experienced by its users. In the context of 5G, QoS encompasses arange of factors, including reliability, availability, latency throughput, andprioritization of traffic. These aspects play a vital role in delivering anoptimal user experience and catering to diverse application requirements.
+  
+## Key Features of 5G QoS
+
+5G technology revolutionizes network management and user experiences through Quality of Service (QoS), introducing groundbreaking capabilities such as Network Slicing, Enhanced Mobile Broadband (eMBB), Ultra-Reliable Low-Latency Communication (URLLC), and Massive Machine-Type Communication (mMTC). Each feature targets specific needs:
+
+### Network Slicing
+
+One of the groundbreaking advancements facilitated by 5G QoS is network slicing. This technology allows service providers to create multiple virtual networks within a single physical infrastructure, each tailored with specific QoS parameters suited for distinct applications or user groups.
+
+### Enhanced Mobile Broadband (eMBB)
+
+Catering to high data rate applications such as 4K video streaming, virtual and augmented reality, and cloud gaming, eMBB under 5G guarantees high throughput to maintain seamless service delivery.
+
+### Ultra-Reliable Low-Latency Communication (URLLC)
+
+URLLC supports applications requiring extremely low latency and high reliability, such as autonomous vehicles and remote surgeries, showcasing 5G's capability to support critical real-time operations.
+
+### Massive Machine-Type Communication (mMTC)
+
+5G enhances the IoT ecosystem by optimizing resource allocation for a vast number of low-power, low-data-rate devices, facilitating advancements in smart cities, smart homes, and industrial IoT.
+
+## 5G QoS Framework
+
+5G networks implement an intricate QoS framework that is pivotal for managing service quality across both control and user plane aspects.
+
+### Control Plane
+
+Central to the control plane are the Access and Mobility Management Function (AMF) and the Authentication Server Function (AUSF). The AMF oversees device connectivity and mobility within the network, initiating UE registration and coordinating with the AUSF to authenticate devices via SIM card verification. This interaction ensures that all devices have valid subscriptions and are properly managed within the network.
```

Comment:
The Control Plane indicates all of the network functions except for UPF.
This part of the description can not let readers understand how it works (e.g. handling the UE registration needs other NFs to get involved).
Moreover, You should focus on explaining How NFs achieve the QoS in the 5G system.